### DISCUSSION TITLE
Excel comparando pagamentos de Impostos para pessoa física e pessoa jurídica

### DISCUSSION THREAD

**Discussion by sibelius** [DyegoCosta/trabalhando-remoto#181] [General]
Seria legal um excel para comparar os impostos pagos em cada uma das modalidades

- [ ] pessoa física com **carne leão**
- [ ] pessoa jurídica com **Simples**
- [ ] pessoa jurídica com **Lucro presumido**

**Comment by vanessametonini** [comment]
Um excel não né, vamos fazer em HTML! haha, mas boa ideia

**Comment by fabioluiz1** [comment]
Segue uma calculadora para Carnê Leão: https://docs.google.com/spreadsheets/d/10nzegN8SaWoa89fnjW6qIx665T5IG5DzG252VpZ4kRI/edit?usp=sharing

Incluí também o cálculo da IRPF no ano seguinte pois, se o valor pago via carnê Leão exceder o imposto devido no ultimo ano, haverá saldo a restituir. 

Para um faturamento de R$ 10000,00 mensais, apesar da alíquota efetiva mensal ser 17,76%, ao considerar a restituição, a alíquota efetiva anual cai para 14,97% (desconsiderei a inflação, pois a restituição sofre correção monetária).

Notar que quando maior o faturamento, maior será o imposto devido e menor a restituição, e mais a alíquota efetiva se aproximará da alíquota máxima para pessoa física, que é de 27,5%.

**Comment by GustavoNunes** [comment]
Olá pessoal! @fabiolnm, obrigado pela planilha!

Isso me gerou uma dúvida. Comecei a trabalhar para o exterior recentemente (sou desenvolvedor de software), e sempre acreditei que trabalhar como PJ valeria mais a pena.

Porém, usando esse faturamento de R$ 10.000,00 como exemplo, vejo no Anexo V do Simples Nacional que a alíquota de imposto seria de 17,50%, enquanto que como pessoa física no Carne Leão a alíquota efetiva é menor (14,97% no exemplo acima).

Me parece que é mais vantajoso, nesse caso, trabalhar como pessoa física. Ainda mais considerando a burocracia de abrir empresa, ter contador, emitir NFs, etc. 
Alguém pode confirmar se meu pensamento é valido ou se estou esquecendo algo desse cenário?

Agradeco desde já a todos, principalmente ao @DyegoCosta pelo repositório.

**Comment by rubemz** [comment]
@fabiolnm Muito boa a planilha! Parabéns, muito útil!

No caso de PJ, não deveria também incluir os custos com um contador? Já que no carnê leão a pessoa mesmo pode declarar e pagar.

**Comment by fabioluiz1** [comment]
Eu tenho PJ, principalmente para projetos locais - minha percepção é de ser mais fácil conseguir projetos em empresas nacionais via PJ, pela facilidade para o empregador. 

Porém, conforme observado pelo @rubemz, o custo do contador morde uma boa fatia do bolo, principalmente se vc não possui sócios para 'dividir a conta'. 

Ao somar todos os custos:
* mensalidade de conta bancária PJ, 
* contador - muitos de meus colegas com PJ pagam meio salário mínimo, o que é bem pesado
* custos esporádicos - alvará e certificados digitais, por exemplo
* Estados / municípios falidos reajustando impostos (há poucos meses o município retirou uma isenção que fez triplicar o valor que eu pago de ISS).
* custo de abertura (caso ainda não a tenha aberto) / fechamento da empresa (caso vc desista após os primeiros anos).

Quando vc divide pelos meses do ano para chegar ao custo mensal, pode-se chegar a até 20% do faturamento. 

Pode ser vantajoso em alguns casos, com o ônus de vc precisar de uma estrutura bancária / contábil e aprender o básico para lidar com isso.

Cheguei a perguntar ao meu contador sobre os requisitos para desenvolver para o exterior via PJ, e ele mencionou sobre mais burocracias, por exemplo:

* SISCOMEX / SISCOSERV
* Tradução juramentada de contrato
* Custos adicionais de Invoices
* Custos bancários por transferência

O Carnê Leão pode ser uma boa opção para não ter que lidar com toda essa burocracia.

**Comment by DyegoCosta** [comment]
Para quem está buscando uma alternativa mais em conta para seu contador: 

https://www.contabilizei.com.br/bem-vindo/?ref=c74999ca3f962678ee774f2b8f4075a4&utm_source=plataforma

**Comment by vanessametonini** [comment]
Eu uso a contabilizei, é muito boa!

**Comment by hjemmel** [comment]
@fabiolnm Essa planilha está certa ? O INSS no carne leão não seria 20% do valor recebido ?

http://www.revitax.com.br/irpf-2016-x-profissional-liberal-novas-obrigacoes-com-o-leao/

**Comment by fabioluiz1** [comment]
Boa pergunta. O melhor texto que consegui achar até o momento é este:
http://odia.ig.com.br/noticia/economia/2016-01-12/contribuicao-para-inss-fica-mais-alta.html

Note que o texto menciona 20% sobre o salário-contribuição, cujo piso é o salário mínimo, e cujo teto é definido anualmente.

**Comment by fabioluiz1** [comment]
Na dúvida, o ideal é perguntar a um contador qual o correto, de preferência conseguindo a referência da portaria que define essa regra.

**Comment by fabioluiz1** [comment]
Uma última contribuição sobre o Carnê-Leão: se trabalhar em casa, pode-se usar o livro caixa, para deduzir algumas despesas:

<img width="611" alt="screen shot 2017-03-13 at 23 13 47" src="https://cloud.githubusercontent.com/assets/508671/23882992/004e7224-0843-11e7-98bc-368312724ede.png">

Fonte: Documentação (Menu Ajuda) do Programa do Carnê Leão 2017.

**Comment by fabioluiz1** [comment]
Finalizando a dúvida sobre o INSS, a alíquota de 20% incide sobre o salário-de-contribuição:

http://www.previdencia.gov.br/servicos-ao-cidadao/todos-os-servicos/gps/forma-pagar-codigo-pagamento-contribuinte-individual-facultativo/

http://www.previdencia.gov.br/servicos-ao-cidadao/todos-os-servicos/gps/tabela-contribuicao-mensal/

**Comment by hjemmel** [comment]
Sim mas o limite desse salário-de-contribuição é mais alto que o salário mínimo não ? O Teto não seria 5.531,31 ?

http://www.previdencia.gov.br/servicos-ao-cidadao/todos-os-servicos/gps/tabela-contribuicao-mensal/

**Comment by fabioluiz1** [comment]
Até onde vai meu conhecimento, é uma escolha pessoal - se a pessoa escolher pagar mais, respeitando o piso e o teto permitidos, em tese terá direito a uma aposentadoria maior.

Esta thread tem inclusive uma tentativa de explicar qual será o valor da aposentadoria:
https://jus.com.br/duvidas/254374/compensa-pagar-valor-maior-inss-quando-a-aposentadoria-se-aproxima

No mais, vale a pena se cadastrar https://servicos.inss.gov.br/central/index.html para ter acesso ao extrato de contribuições, e talvez por lá se consiga mais informações.

**Comment by hjemmel** [comment]
Bom eu tinha lido coisa diferente é para ser com base nos ganhos do contribuinte individual, porém vou seguir sua sugestão e vou procurar um contador para me informar mais certo.

**Comment by strongwinderic** [comment]
Boa noite pessoal!
Recolher o INSS é obrigatório para pessoa física que trabalha remoto para o exterior?

**Comment by vanessametonini** [comment]
Não sei como tá o carnê leão hoje, se é tudo embutido, mas qdo fiz pelo carnê leão em 2015, não tinha uma guia separada para o INSS, era só guia da DARF sob o valor declarado...

[edit]
Eu não paguei INSS neste período com o carnê leão. Quando vc é autônomo vc nao é obrigado a pagar INSS. Acho que só é quando vc é celetista (CLT) ou pj (pelo prolabore).


Em IMHO, até vale a pena pagar carnê leão, na restituição do IR volta uma bolada. Pelo menos foi oq aconteceu cmg, nao sei como foi com vcs xD

**Comment by fabioluiz1** [comment]
Também não sei se é obrigatório recolher o INSS, mas vc pagar, entra como dedução no valor do imposto a ser pago no carnê-leão. 

Diminui um pouco o imposto pago no mês, e ainda pode ajudar a melhorar um pouquinho uma possível restituição do IRPF. 

Vale à pena simular usando o programa do Carnê-Leão e decidir se vale a pena pro seu caso.

**Comment by danielbathke** [comment]
@strongwinderic, não é obrigatório.

Agora, se vc já trabalhou aqui no Brasil e já recolheu alguma coisa de INSS, se você não contribuir por um tempo (não sei quanto), você vai acabar por "perder" esse "benefício".

Agora, até onde INSS é um benefício ... vai de cada um.

Abraço.

**Comment by pablo-sg-pacheco** [comment]
Uma notícia boa/ruim.

A partir de 2018 algumas regras vão mudar. Pelo que entendi, se o gasto com folha de pagamento for menor que 28% do faturamento, a empresa pode ficar enquadrada no anexo 3, pagando apenas 6%, em vez de 17% do anexo V. 

Pra que você tenha uma folha de pagamento, basta que vc se pague um pró-labore com um valor de pelo menos 28% do faturamento. Mas na minha opinião isso só vale a pena se vc criar esse pro labore no valor de até 1900 reais, pra ficar isento na tributação de pessoa física.

`Dica importante: Quanto maior for a folha de pagamento da empresa prestadora de serviços, menor a alíquota. Isso quer dizer que mesmo as atividades que em teoria pagam mais impostos podem ser enquadradas ainda no anexo III. Para isso, a razão entre o valor da folha salarial e a receita bruta deve ser igual ou maior que 28%.`
http://www.administradores.com.br/artigos/empreendedorismo/novo-simples-nacional-20172018-entenda-as-7-principais-mudancas-e-como-elas-irao-impactar-a-vida-das-empresas/98999/
